packages/EasyErrorHandler/src/Bridge/Symfony/Providers/ErrorCodesByEnumProvider.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Symfony\Providers;
+
+use EonX\EasyErrorHandler\Annotations\AsErrorCodes;
+use EonX\EasyErrorHandler\Interfaces\ErrorCodesProviderInterface;
+use PhpParser\Error;
+use PhpParser\Node\Stmt\Enum_;
+use PhpParser\Node\Stmt\Namespace_;
+use PhpParser\Parser;
+use PhpParser\ParserFactory;
+use ReflectionClass;
+
+final class ErrorCodesByEnumProvider implements ErrorCodesProviderInterface
```

Comment:
Let's move it from the Symfony bridge to the common level, we can easily reuse it for Laravel as well.

Reply:
ErrorCodesFromEnumsProvider moved to common folder